### DISCUSSION TITLE
How to show/hide tooltips?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1464] [Q&A]
Hi

How can I hide a tooltip so that it does not show even when I hover.

https://play.tailwindcss.com/atfeD9g4Mi

I'd like to hide and show my tooltips based on some condition. Adding the hidden class on the tooltip also hides the button.

Thank you so much

**Comment by fini** [comment]
In react I use the following function to omit the tooltip if there is no tooltip:

```
const toolTip = (tipText, toolTipChildren) =>
  tipText ? (
    <div className="tooltip tooltip-bottom tooltip-info" data-tip={tipText}>
      {toolTipChildren}
    </div>
  ) : (
    toolTipChildren
  )

export default toolTip
```
